AsyncDisplayKitTests/ASDisplayNodeTests.m

```diff
@@ -1989,7 +1980,7 @@ - (void)testDidEnterFetchDataIsCalledWhenNodesEnterFetchDataRange
 
 - (void)testDidExitFetchDataIsCalledWhenNodesExitFetchDataRange
 {
-  ASTestDisplayNode *node = [[ASTestDisplayNode alloc] init];
+  ASTestDisplayNode *node = [[[ASTestDisplayNode alloc] init] autorelease];
```

Comment:
Thanks for adding all these.  Standardizing on autorelease is a good move where performance doesn't matter and looking at correctness is the priority.